channels/tsmf/client/gstreamer/tsmf_gstreamer.c

```diff
@@ -750,7 +942,7 @@ BOOL tsmf_gstreamer_ack(ITSMFDecoder* decoder, BOOL (*cb)(void *, BOOL), void *s
 {
 	TSMFGstreamerDecoder* mdecoder = (TSMFGstreamerDecoder *) decoder;
 	DEBUG_TSMF("");
-	mdecoder->ack_cb = cb;
+	mdecoder->ack_cb = NULL;//cb;
 	mdecoder->stream = stream;
```

Comment:
agreed, should probably just remove the whole ack_cb stuff from gstreamer decoder code.